test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -12595,7 +12659,17 @@ def __init__(self, name=None, attrs={}, pbsconf_file=None, diagmap={},
         self.configd = os.path.join(self.pbs_conf['PBS_HOME'], 'mom_priv',
                                     'config.d')
         self.config = {}
-        self.dflt_config = {'$clienthost': self.server.hostname}
+        if self.platform == 'cray' or self.platform == 'craysim':
+            if self.platform == 'cray':
+                alps_client = self.du.which(exe='apbasil')
+            else:
+                alps_client = "/opt/alps/apbasil.sh"
+            self.dflt_config = {'$clienthost': self.server.hostname,
+                                '$vnodedef_additive': 0,
+                                '$alps_client': alps_client,
+                                '$usecp': '*:/home /home'}
```

Comment:
This is needed on Cray having two moms when submitting a job without the qsub -o option.  If a job is submitted from the second mom, which gets executed on the first mom, then we should see the job's output files end up in the user's home directory and the job will not stay in E state and terminate normally.